website/server/models/task.js

```diff
@@ -247,10 +247,10 @@ export let DailySchema = new Schema(_.defaults({
     type: String,
     default: 'weekly',
     enum: [
-      'daily', // Every X Interval
+      'daily', // Every X Intervals
       'weekly', // Certain Days of the Week
       'sinceCompletion', // Every X Interval Since Completion
-      'timesInInterval', // Y Times in X Interval
+      'timesInInterval', // Y Times in X Intervals
```

Comment:
What's the question? Valid setups for this might include:

3 times in 1 Month
5 times in 3 Weeks
1 time in 2 Years
1 time in 1 Day (equivalent to current default active-every-day Dailies)